.gitignore

```diff
@@ -463,11 +463,6 @@ tags
 
 ### VisualStudioCode ###
 .vscode/*
-!.vscode/settings.json
```

Comment:
why are these removed?

Reply:
I guess I generally haven't found checking in anything under `.vscode` to be useful, especially in projects where developers aren't all necessarily using the same IDE. But I don't really have strong feelings on it, and I'm happy to revert this and check in my `settings.json` instead.

Reply:
Oh okay, it's definitely the case that I don't want you committing IDE settings. Weird that this is even in here, does this imply that the gitignore.io wants people committing this?

Reply:
I haven't done an immense amount of research on it, but I think it's one of those things where reasonable people can have differing opinions. Like, my `settings.json` has stuff in it that says "this project uses `pytest` instead of `unittest`" which could be beneficial for everyone, but it _could_ also have stuff in it that's like "I want my editor to have a larger font for this project". So I err on the conservative side and leave it out generally.

Reply:
yeah, for sure this stuff doesn't belong in version control